### ISSUE TITLE
No error thrown when parsing a pylintrc file that enables a rule that does not exist

### ISSUE THREAD

**Issue by eli-fine-res** [pylint-dev/pylint#8911]
### Bug description

If there's an attempt to enable a rule in the `pylintrc` file, but there's a typo or the rule is just named wrong, then pylint proceeds without any issue.   It should throw an error if there's a listed rule that doesn't actually exist right?

to reproduce, set pylintrc:
`enable=not-a-real-rule`

### Configuration

```ini
enable=not-a-real-rule
```

### Command used

```shell
`pre-commit run -a`


    - repo: local
      hooks:
          - id: pylint
            name: pylint-src
            entry: pylint
            language: system
            files: '.*\.py$'
            exclude: "tests/.*"
            verbose: true
            args:
                - --rcfile=src/pylintrc
```
```


### Pylint output

```shell
pylint-src...............................................................Passed
- hook id: pylint
- duration: 6.92s

------------------------------------
Your code has been rated at 10.00/10
```

### Expected behavior

"Error parsing pylintrc: The rule 'not-a-real-rule' was attempted to be enabled on line ##, but there was no rule found in pylint or any plugins with that name"

### Pylint version

```shell
root@4daf43a56087:/home# pylint --version
pylint 2.17.4
astroid 2.15.6
Python 3.9.13 (main, Aug  2 2022, 11:27:52) 
[GCC 8.3.0]
```

### OS / Environment

Ubuntu 20

**Comment by jacobtylerwalls** [comment]
Thanks. Can you share the contents of `src/pylintrc`? When I run your example, I get:

```sh
************* Module /Users/jwalls/pylint/pylintrc
pylintrc:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'not-a-real-rule' (unknown-option-value)
```

**Comment by eli-fine-res** [comment]
hmm, thanks for taking a look.  this is intriguing.  so the bug is in my pylintrc somewhere.


I adjusted to

```
[MASTER]



[MESSAGES CONTROL]
enable=not-a-real-rule

```

and then sure enough:
```
pylint-src...............................................................Failed
- hook id: pylint
- duration: 6.08s
- exit code: 30

************* Module src/pylintrc
src/pylintrc:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'not-a-real-rule' (unknown-option-value)
```


well that's encouraging to know that it is feature of pylint.  now to try and figure out what's wrong with the config file.   side note---I've tried lots of times to find some really good documentation about how to set up a pylintrc file, but I've never been successful

**Comment by jacobtylerwalls** [comment]
I'm wondering if it's your pre-commit hook filtering on .py files only. Try including the rcfile in that regex.

**Comment by jacobtylerwalls** [comment]
> I've tried lots of times to find some really good documentation about how to set up a pylintrc file, but I've never been successful

Yours looks good! Hopefully it's just the pre-commit action 🤞

**Comment by jacobtylerwalls** [comment]
Ah, sorry, didn't scroll up. What if you remove `--jobs=0`? We've been dealing with lots of subtle incompatibilities with multiprocessing.

**Comment by eli-fine-res** [comment]
commenting out the `jobs=0` line did the trick, thanks!   Is there a label you can apply to this issue to recategorize it as a multiprocessing bug?


```
pylint-src...............................................................Passed
- hook id: pylint
- duration: 6.61s

************* Module src/pylintrc
src/pylintrc:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'not-a-real-rule' (unknown-option-value)

-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 8.32/10, +1.68)

************* Module src/pylintrc
src/pylintrc:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'not-a-real-rule' (unknown-option-value)

-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 9.09/10, +0.91)
```

**Comment by eli-fine-res** [comment]
for the record, I'm personally fine for now just disabling multiprocessing.  I just didn't want this bug discovery to end up completely forgotten as people eventually start to tackle the multiprocessing issues
